### ISSUE TITLE
[Feature] Command line test

### ISSUE THREAD

**Issue by madmath03** [humhub/humhub#3621]
### Describe the solution you'd like
The `InstallController` is currently used for integration tests, but it could be used for command line install of HumHub (already the case in [this docker image)](https://github.com/mriedmann/humhub-docker#humhub_auto_install).

To do that, the installer would need some external parameters (with the same default value as now) for the following fields:
* instance settings
https://github.com/humhub/humhub/blob/f072f95321e1fab18e8533bf4e80860cc24866f6/protected/humhub/modules/installer/commands/InstallController.php#L36-L39
* admin username
https://github.com/humhub/humhub/blob/f072f95321e1fab18e8533bf4e80860cc24866f6/protected/humhub/modules/installer/commands/InstallController.php#L43
* admin email
https://github.com/humhub/humhub/blob/f072f95321e1fab18e8533bf4e80860cc24866f6/protected/humhub/modules/installer/commands/InstallController.php#L44
* admin firstname and lastname
https://github.com/humhub/humhub/blob/f072f95321e1fab18e8533bf4e80860cc24866f6/protected/humhub/modules/installer/commands/InstallController.php#L52-L53
* admin password
https://github.com/humhub/humhub/blob/f072f95321e1fab18e8533bf4e80860cc24866f6/protected/humhub/modules/installer/commands/InstallController.php#L58

### Describe alternatives you've considered

After running installer command, updating the databases entries manually :confused:

**Comment by luke-** [comment]
Good point, the upcoming 1.4 will provide a command line controller to manage settings (e.g. name)

https://github.com/humhub/humhub/blob/develop/protected/humhub/commands/SettingsController.php

**Comment by acs-ferreira** [comment]
@luke- is the a changelog for the upcoming v1.4? https://github.com/humhub/humhub/blob/develop/protected/humhub/docs/CHANGELOG.md is not up-to-date.

Thanks.

**Comment by luke-** [comment]
There is a separate CHANGELOG for easier merging.

https://github.com/humhub/humhub/blob/develop/protected/humhub/docs/CHANGELOG_DEV.md

**Comment by madmath03** [comment]
Hi

Just curious to know if this has been implemented with 1.4 or not.

**Comment by luke-** [comment]
It's possible to manage settings from command line:
https://docs.humhub.org/docs/admin/console#settings

Admin user creation is still missing
